src/sentry/static/sentry/app/components/forms/multiSelectField.jsx

```diff
@@ -0,0 +1,78 @@
+import React from 'react';
+import PropTypes from 'prop-types';
+import Select from 'react-select';
+import styled from 'react-emotion';
+
+import InputField from './inputField';
+
+export default class MultiSelectField extends InputField {
+  static propTypes = {
+    options: PropTypes.array.isRequired,
+    onChange: PropTypes.func,
+    value: PropTypes.any,
+  };
+
+  constructor(props) {
+    super(props);
+    this.state = {
+      selectedValues: [],
+    };
+  }
+
+  handleChange = value => {
+    this.setState({selectedValues: value}, () => {
+      if (typeof this.props.onChange === 'function') {
+        this.props.onChange(value);
+      }
+    });
+  };
+
+  renderArrow = () => {
+    return <span className="icon-arrow-down" />;
```

Comment:
nice

Reply:
maybe this should be the inlineSvg version tho?

Reply:
I think we should keep this one to match the other dropdowns, the svg version looks different